🚸(edxapp) provision demo test and staff users

rmoch opened the pull request.

sampaccoud commented on group_vars/all/main.yml:
```diff
@@ -114,3 +114,34 @@ openshift_volumes:
+    hashed_password: "{{ demo_hashed_password }}"
+    is_staff: true
+    is_superuser: true
+demo_test_staff_users: "{{ demo_test_users }} + {{ demo_staff_users }}"
```
A better name may be `demo_all_users` ??

rmoch replied:
I was reusing edX variable names

sampaccoud commented on templates/openshift/edxapp/job/import_demo_course.yml.j2:
```diff
@@ -29,7 +29,17 @@ spec:
-                python manage.py cms import /edx/app/edxapp/data /tmp/edx-demo-course*
+                python manage.py cms import /edx/app/edxapp/data /tmp/edx-demo-course* &&
+                {% for user in demo_test_staff_users %}
+                python manage.py lms manage_user {{ user.username}} {{ user.email }} --initial-password-hash {{ user.hashed_password | quote }} {% if user.is_staff %}--staff{% endif %} {% if user.is_superuser %}--superuser{% endif %}
```
add indentation as discussed with @jmaupetit ?

sampaccoud commented on group_vars/all/main.yml:
```diff
@@ -114,3 +114,34 @@ openshift_volumes:
   - openshift/richie/pvc/media.yml
   - openshift/richie/pvc/static.yml
+
+  # default users for development environment
```
it should be `development` AND `feature` environments

jmaupetit commented:
As discussed with @sampaccoud, one way to improve handling course import and avoid polluting the main configuration is:

* create a `groups_vars/all/fixtures.yml` that contains users data,
* add a `load_fixtures.yml` playbook that loads these variable definitions and create _ad hoc_ job objects.

sampaccoud committed:
🚸(edxapp) Add playbook to run job to load fixtures

Import demo course and create demo users. We mount
a python script file in configmap and pass it to
the Django shell.

jmaupetit commented on README.md:
````diff
@@ -79,8 +79,27 @@ $ bin/init
+completed successfully, you can create a demo course and some users by running:
+
+```bash
+$ bin/run load_fixtures.yml
````
```
bin/ansible-playbook load_fixtures.yml
```

jmaupetit reviewed (approved):
:tada:
